test-framework/common/src/main/java/io/quarkus/test/common/QuarkusTestResourceLifecycleManager.java

```diff
@@ -28,7 +28,7 @@ public interface QuarkusTestResourceLifecycleManager {
     /**
      * Start the test resource.
      *
-     * @return A map of system properties that should be set for the running test
+     * @return A map of configuration properties that should be set for the running test
```

Comment:
Isn't it "will be set" instead of "should be"

Reply:
Agreed, `will be set` is more clear in the method's intention

Reply:
```suggestion
     * @return A map of configuration properties that will be set for the running test
```